src/OrleansProviders/Streams/Common/PooledCache/PooledQueueCache.cs

```diff
@@ -77,7 +77,7 @@ public class PooledQueueCache<TQueueMessage, TCachedMessage>: IPurgeObservable<T
         /// <param name="logger"></param>
         /// <param name="cacheMonitor"></param>
         /// <param name="cacheMonitorWriteInterval">cache monitor write interval</param>
-        public PooledQueueCache(ICacheDataAdapter<TQueueMessage, TCachedMessage> cacheDataAdapter, ICacheDataComparer<TCachedMessage> comparer, Logger logger, ICacheMonitor cacheMonitor = null, TimeSpan? cacheMonitorWriteInterval = null)
+        public PooledQueueCache(ICacheDataAdapter<TQueueMessage, TCachedMessage> cacheDataAdapter, ICacheDataComparer<TCachedMessage> comparer, Logger logger, ICacheMonitor cacheMonitor, TimeSpan? cacheMonitorWriteInterval)
```

Comment:
Make param cacheMonitor and cacheMonitorInterval non-optional, since now all three stream providers has monitoring add to them, so I don't want user take the delusion that monitoring is optional on some stream providers. 

But user can still set those two values to null in testing scenario